src/Gun.java

```diff
@@ -0,0 +1,11 @@
+/**
+ * Created by daveoffice on 2/7/17.
+ */
+public class Gun {
+    String maker;
+    String cycleoffire;
+    int magazinecapacity;
+    int caiiber;
+    boolean isPistol;
+
```

Comment:
variables follow the naming convention of first word starts with a lowercase letter the second word starts with capital letters.  So line 9 is done correctly.  Line 7 should be magazineCapacity. you should follow that format in the future. Overall good job.